fertility/orderly-web.yml

```
@@ -0,0 +1,133 @@
+## arrange a proxy on this network, or use dev_mode in the web section
+## below.
+network: fertility_nw
+
+## Names of the docker volumes to use:
+##
+## orderly: stores the orderly archive
+## proxy_logs: stores logs from the reverse proxy (only used if proxy is given)
+## css: stores compiled css for the web app (only needed if custom sass variables
+## are given as sass_variables in web section below)
+##
+## (More volumes are anticipated as the tool develops)
+volumes:
+  orderly: fertility_orderly_volume
+  proxy_logs: fertility_web_proxy_logs
+  css: fertility_web_css
+  redis: fertility_redis_data
+
+## Orderly configuration
+orderly:
+  ## Image to use for orderly.  This should build off of
+  ## vimc/orderly.server but can be extended to use whatever packages
+  ## you need.  The components repo, image and tag will be assembled as
+  ## <repo>/<name>:<tag> for the full docker image reference.
+  image:
+    repo: mrcide
+    name: fertility-orderly
+    worker_name: fertility-orderly
+    tag: main
+  initial:
+    source: clone
+    url: git@github.com:osymandius/fertility_orderly
+  workers: 1
+  env:
+    ORDERLY_API_SERVER_IDENTITY: main
+    ## VIMC-3692
+    VAULT_AUTH_GITHUB_TOKEN: mustexist
+    MC_CORES: 4
+    SHAREPOINT_USERNAME: VAULT:secret/hiv-orderly/sharepoint:username
+    SHAREPOINT_PASS: VAULT:secret/hiv-orderly/sharepoint:pass
+    ADR_KEY: VAULT:secret/hiv-orderly/adr:key
+  ssh:
+    public: VAULT:secret/hiv-orderly/deploy-keys/mrc-ide/naomi-orderly:pub
+    private: VAULT:secret/hiv-orderly/deploy-keys/mrc-ide/naomi-orderly:key
+
+## Api and Website configuration
+web:
+  ## Image to use for the web components.  As for the orderly
+  ## configuration these will be assembled as <repo>/<name>:<tag>.  In
+  ## addition, a second image with the database migration support will
+  ## be used as <repo>/<migrate>:<tag>. It is not expected (unlike
+  ## the orderly image) that the 'name' and 'migrate' will need to be
+  ## changed often, and it is expected that a web image will go
+  ## together with corresponding migration and admin images. If custom sass
+  ## variables are provided below as sass_variables, a third image
+  ## for generating css is required.
+  image:
+    repo: vimc
+    name: orderly-web
+    tag: master
+    migrate: orderlyweb-migrate
+    admin: orderly-web-user-cli
+    css-generator: orderly-web-css-generator
+  ## Public-facing url for the whole web service, including protocol
+  ## (ideally https://), hostname and port (if not using standard
+  ## ports).  Here, we're going to use the same as for the proxy but
+  ## if you are using an external proxy then you'd use its hostname
+  ## and port.
+  url: https://hiv-orderly.dide.ic.ac.uk/fertility
+  ## If dev_mode is true then the port is exposed to the host (as
+  ## plain http).  Do not use in production.  The port is attached
+  ## only to the localhost and will not be available from other
+  ## machines
+  dev_mode: true
```

I think we can remove this now

```suggestion
  dev_mode: false
```